tests/_path.py

```diff
@@ -67,7 +67,8 @@ def build(
 @functools.singledispatch
 def create(content: Union[str, bytes, FilesSpec], path):
     path.mkdir(exist_ok=True)
-    build(content, prefix=path)  # type: ignore[arg-type]
+    # Mypy only looks at the signature of the main singledispatch method. So it must contain the complete Union
+    build(content, prefix=path)  # type: ignore[arg-type] # python/mypy#11727
```

Comment:
This file is copied from upstream (jaraco.path). These changes will get overridden the next time this code is refreshed. Since they're only comments, let's defer to the comment above to track all issues with this code upstream. (I'll take care of it)